### PR TITLE
Updating training docs with adding option to score calibration and automatic label encoding

### PR DESCRIPTION
**Pull request by sunishsheth2009**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -511,6 +511,15 @@ Below are all the possible options and full reference guide for different config
     estimator_method: estimator_fn
     ```
 
+  - `calibrate_proba`: string. Optional. 
+  Probability calibration can be enabled by passing in the method to use for calibration.
+  Values could either be ‘sigmoid’ or ‘isotonic’.
```
Comment (jinzhang21) [inline README.md:516]: Add: `For more information, see sklearn [CalibratedClassifierCV](https://scikit-learn.org/stable/modules/generated/sklearn.calibration.CalibratedClassifierCV.html)`

**Review by jinzhang21 — APPROVED** [review]
